fakestorage/server.go

```diff
@@ -73,6 +73,10 @@ func NewServerWithHostPort(objects []Object, host string, port uint16) (*Server,
 	})
 }
 
+type EventManagerOptions = notification.EventManagerOptions
+
+type EventNotificationOptions = notification.EventNotificationOptions
```

Comment:
Can you remove this since it's unused?

Reply:
@fsouza I'm not sure if it can be removed.  I was starting the fake server like so:

```go
    return fakestorage.NewServerWithOptions(fakestorage.Options{                                                                                                                                                                                                       
        Scheme:      "http",                                                                                                                                                                                                                                           
        NoListener:  false,                                                                                                                                                                                                                                            
        Host:        host,                                                                                                                                                                                                                                             
        Port:        port,                                                                                                                                                                                                                                             
        PublicHost:  hostPort,                                                                                                                                                                                                                                         
        StorageRoot: storageRoot,                                                                                                                                                                                                                                      
        EventOptions: fakestorage.EventManagerOptions{                                                                                                                                                                                                                 
            ProjectID: projectID,                                                                                                                                                                                                                                      
            TopicName: topicName,                                                                                                                                                                                                                                      
            NotifyOn: fakestorage.EventNotificationOptions{                                                                                                                                                                                                            
                Finalize: true,                                                                                                                                                                                                                                        
                Delete:   true,                                                                                                                                                                                                                                        
            },                                                                                                                                                                                                                                                         
        },                                                                                                                                                                                                                                                             
    })
```

Like `EventManagerOptions`, `EventNotificationOptions` is also defined in an internal package.

Reply:
ohh ok, sorry about that. let's bring it back then. do you want to open a PR or would you rather have me do it? I can push a release right after

Reply:
I think that you've handled it already.  Thanks :)